content/en/docs/concepts/containers/images.md

```diff
@@ -157,6 +157,30 @@ that Kubernetes will keep trying to pull the image, with an increasing back-off
 Kubernetes raises the delay between each attempt until it reaches a compiled-in limit,
 which is 300 seconds (5 minutes).
 
+## Serial and parallel image pulls
+
+By default, kubelet pulls images serially. In other words, kubelet will send only
+one image pull request to the image service at a time. Other image pull requests
+have to wait until the one being processed is complete.
+
+If you would like to enable parallel image pulls, you can set the field
+`serializeImagePulls` to false in kubelet configuration. With `serializeImagePulls`
+set to false, image pull requests will be sent to the image service immediately,
+and multiple images will be pulled at the same time.
```

Comment:
please note the difference of parallel image pulls from different pods. Images from the same pods are never pulled in parallel